Automattic/wp-calypso
# SEO: Migrate Image to Webpack

#### Changes proposed in this Pull Request

This migrates the one image in `/seo` to Webpack. For details on the reasoning behind this, see #36850.

#### Testing instructions

There should only be one use of this image: https://github.com/Automattic/wp-calypso/search?q=page-title.svg&unscoped_q=page-title.svg

You can find it by visiting `/marketing/traffic/site` on a site with a Business Plan. Verify that the image is still there and no different. 

<img width="1149" alt="Screenshot 2019-11-17 at 19 44 21" src="https://user-images.githubusercontent.com/43215253/69012950-daccd480-0972-11ea-9762-41c8f24fc665.png">
